src/test/CMakeLists.txt

```diff
@@ -161,9 +161,10 @@ if(ENABLE_WALLET)
   add_subdirectory(${PROJECT_SOURCE_DIR}/src/wallet/test wallet)
 endif()
 
-if(WITH_MULTIPROCESS)
+if(ENABLE_IPC)
   target_link_libraries(bitcoin_ipc_test
     PRIVATE
+      Boost::headers
```

Comment:
1b3613505deb4baeacc3c4c2fc91ccca1a8ecec5

Usually, external dependencies follow the internal ones.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/31741#discussion_r1948687873


> [1b36135](https://github.com/bitcoin/bitcoin/commit/1b3613505deb4baeacc3c4c2fc91ccca1a8ecec5)
> 
> Usually, external dependencies follow the internal ones.

Makes sense, fixed this.